editor/enable-tracking-prompt/index.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from 'i18n';
+import { Button } from 'components';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import { removeNotice } from '../actions';
+
+function EnableTrackingPrompt( { dispatchRemoveNotice } ) {
+	function dismissTrackingPrompt( result ) {
+		window.setUserSetting( 'gutenberg_tracking', result );
+		dispatchRemoveNotice( 'notice:enable-tracking-prompt' );
+	}
+
+	return (
+		<div className="enable-tracking-prompt">
+			<div className="enable-tracking-prompt__message">
+				{ __( 'Can Gutenberg collect data about your usage of the editor?' ) }
+				<div className="enable-tracking-prompt__buttons">
+					<Button
+						isPrimary
+						isSmall
+						onClick={ () => dismissTrackingPrompt( 'on' ) }
+					>
+						{ __( 'Yes' ) }
+					</Button>
+					<Button
+						isSecondary
+						isSmall
+						onClick={ () => dismissTrackingPrompt( 'off' ) }
+					>
+						{ __( 'No' ) }
+					</Button>
+				</div>
+			</div>
+			<div className="enable-tracking-prompt__clarification">
+				{ __( 'Usage data is completely anonymous and will only be used to improve the editor.' ) }
+			</div>
+		</div>
+	);
+}
+
+export default connect(
+	undefined,
+	{ dispatchRemoveNotice: removeNotice }
```

Comment:
An alternative (used elsewhere) would be to avoid destructuring: `props.removeNotice`